experimenter/experimenter/features/manifests/apps.yaml

```diff
@@ -96,6 +96,13 @@ mozillavpn_backend_cirrus:
     name: "mozilla-services/guardian-website"
   fml_path: "nimbus.yaml"
 
+accounts_cirrus:
```

Comment:
I would rename this to `fxa_cirrus` or `firefox_accounts_cirrus` or similar. The name doesn't really matter (only the slug does).